contracts/src/access/ownable_two_step.rs

```diff
@@ -38,14 +38,6 @@ sol! {
         address indexed new_owner
     );
 
-    /// Emitted when ownership gets transferred between accounts.
-    ///
-    /// * `previous_owner` - Address of the previous owner.
-    /// * `new_owner` - Address of the new owner.
-    event OwnershipTransferred(
```

Comment:
Don't forget to update the Events section for [_transfer_ownership](https://github.com/OpenZeppelin/rust-contracts-stylus/pull/404/files#diff-a792a5d83a14094d0fbcbde7b4e41620f927a81aab7154c2e30b2f3e71f4c28eR221) to state that it emits `crate::access::ownable::OwnershipTransferred`

Reply:
Solved.